internal/pkg/deploy/cloudformation/stack/scheduled_job.go

```diff
@@ -90,12 +90,13 @@ func (e errDurationInvalid) Error() string {
 
 // ScheduledJobConfig contains data required to initialize a scheduled job stack.
 type ScheduledJobConfig struct {
-	App           string
-	Env           string
-	Manifest      *manifest.ScheduledJob
-	RawManifest   []byte
-	RuntimeConfig RuntimeConfig
-	Addons        addons
+	App                 string
```

Comment:
nit: maybe we might just as well pass `App *config.Application` like we do in backend / LBWS and RDWS here. same for worker service!